packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -138,13 +144,20 @@ const AccountTransferForm = observer(
         const is_dxtrade_transfer = selected_to.is_dxtrade || selected_from.is_dxtrade;
         const is_derivez_transfer = selected_to.is_derivez || selected_from.is_derivez;
 
+        const is_poa_or_poi_pending_for_mf =
+            (authentication_status?.document_status === 'pending' ||
+                authentication_status?.identity_status === 'pending') &&
+            is_eu;
+
```

Comment:
can we use "getMultipliersAccountStatus" from client.js

Reply:
ah yeah, sure, will check on this 🙏